real_time/arduino_src/radio_buggy_mega/Makefile

```diff
@@ -235,7 +235,9 @@ AVRDUDE_FLAGS += $(AVRDUDE_NO_VERIFY)
 AVRDUDE_FLAGS += $(AVRDUDE_VERBOSE)
 AVRDUDE_FLAGS += $(AVRDUDE_ERASE_COUNTER)
 
-AVRDUDE_A_PORT = /dev/tty.asdf
+#Change this variable to usb port to which arduino is connected
+#It can be checked by using "ls /dev" and looking for "tty.usbxxxxxx####"
+AVRDUDE_A_PORT = /dev/tty.asdf 

```

Comment:
This was meant to force you to specify a port to prevent accidentally programming the wrong device. I guess this should become an error and exit if the variable isn't defined.